css/annotations/popup.scss

```diff
@@ -1,7 +1,8 @@
 // Colors for buttons.
 $button-background-color: #f7f7f7;
 $button-hover-color: #e6ebf5;
-
+$blue: #424BCD;
+$white: #fff;
```

Comment:
While `$white` is fine, `$blue` is not necessarily the best description.

Reply:
is `$dark-blue` ok?